recipes/tixi3/all/conanfile.py

```diff
@@ -0,0 +1,128 @@
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools import files
+from conan import ConanFile
+import os
+import textwrap
+
+required_conan_version = ">=1.45.0"
+
+
+class Tixi3Conan(ConanFile):
+    name = "tixi3"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "A simple xml interface based on libxml2 and libxslt"
+    topics = ("tixi3", "xml")
```

Comment:
```suggestion
    topics = ("xml", "xml2", "xslt")
```
We dont add conan neither the project name as topic.